sorl_watermarker/engines/base.py

```diff
@@ -1,11 +1,12 @@
-from functools import wraps
-
+from django.conf import settings
 from django.contrib.staticfiles.finders import find
+from django.core.exceptions import ImproperlyConfigured
 from sorl.thumbnail.engines.base import EngineBase as ThumbnailEngineBase
 from sorl_watermarker.parsers import parse_geometry
 from sorl_watermarker.conf import settings
-from django.core.exceptions import ImproperlyConfigured
 
+from functools import wraps
+import os
```

Comment:
Also a cool refactoring would be to organize the imports in accordance with PEP 8 recommendations[1] (in respect of the order of imports).
[1] http://dinsdale.python.org/dev/peps/pep-0008/#imports